### DISCUSSION TITLE
Need help train using a custom models using fast-DreamBooth

### DISCUSSION THREAD

**Discussion by juan9999** [TheLastBen/fast-stable-diffusion#1043] [Q&A]
Using latest version of fast-DreamBooth

I'd like to train with a custom model based on 1.5

There doesn't seem to be an option in the notebook to do that. To work around, I've tried to:
- Upload the model I want to use to sd/stable0diffusion-webui/models/stable-diffusion/
- Renaming model to model.ckpt
But the notebook couldn't find

So I DL 1.5 (thinking I'd just replace the model) but I can't find it on my google drive.

What do I need to do to train using a custom model?

**Comment by TheLastBen** [comment]
> There doesn't seem to be an option in the notebook to do that

in the dreambooth model download cell, you have all the options there, including using a custom model

**Comment by juan9999** [comment]
My bad! Got it working once I figured out I had to add /MyDrive/ in the path after /content/gdrive

May be helpful to have that as an example in the comments.

Also, are these normal training speeds for a 1.5 512 model on a T4?

Progress:|████                     | 15% 455/3000 [07:33<41:39,  1.02it/s, loss=0.0912, lr=1.71e-6]

Thanks again!

**Reply by TheLastBen** [reply]
1it/s is the normal for 512 yes

**Reply by SorenTruelsen** [reply]
Looks like a very normal time for me
